lighting/raymarch.glsl

```diff
@@ -38,28 +39,52 @@ license:
 
 #ifndef ENG_RAYMARCHING
 #define ENG_RAYMARCHING
-vec4 raymarch(vec3 camera, vec3 ta, vec2 st) {
+
+vec4 raymarch( vec3 camera, vec3 ta, vec2 st,
+    out float eyeDepth, out vec3 worldPos, out vec3 worldNormal) {
+
     mat3 ca = RAYMARCH_CAMERA_MATRIX_FNC(camera, ta);
     float fov = 1.0/tan(RAYMARCH_CAMERA_FOV*PI/180.0/2.0);
+    vec3 cameraForward = normalize(ta - camera);
     st.x = 1.0 - st.x;
 
 #if defined(RAYMARCH_MULTISAMPLE)
     vec4 color = vec4(0.0);
+    eyeDepth = 0;
```

Comment:
change to `0.0` older versions of GLSL don't do implicit casting

Reply:
Argh, sorry. Keep forgetting about this, as I'm testing under OpenGL 4.5 and Unity, but not WebGL.

Reply:
All good! I'm sorry to bring it up later. I will do my part better and watch for those running against OpenGL ES 2.0 (you can't get older than that : )

Reply:
Sounds good! Just PRed a fix.